CHANGELOG.adoc

```diff
@@ -6,6 +6,7 @@
 
 - [ADR-039] Provide a variable to detect the environment
 - [ADR-040] Add support for a post selection
+- [ADR-041] Do not considering the source and target connections as routing points anymore
```

Comment:
Typo: "Do not consider" or "Stop considering", but not "Do not considering".

Reply:
Done